gluten-substrait/src/main/scala/org/apache/gluten/backendsapi/SparkPlanExecApi.scala

```diff
@@ -72,6 +73,8 @@ trait SparkPlanExecApi {
       child: SparkPlan): ProjectExecTransformer =
     ProjectExecTransformer.createUnsafe(projectList, child)
 
+  def genSparkPartialProjectColumnarExec(original: ProjectExec): GlutenPlan = null
```

Comment:
Instead of adding code to `OffloadSingleNode`, can we just add a new rule to [Velox's rule list](https://github.com/apache/incubator-gluten/blob/5790c30959dad59284a5e50015d94912b79f1b23/backends-velox/src/main/scala/org/apache/gluten/backendsapi/velox/VeloxRuleApi.scala#L47), then remove this API from backend API?